src/web-capabilities.ts

```diff
@@ -77,4 +77,13 @@ export class WebCapabilities {
     }
     return CapabilityState.CAPABLE;
   }
+
+  /**
+   * Retrieves the current CPU information of the system.
+   *
+   * @returns An object containing details about the CPU.
+   */
+  static getCpuInfo(): CpuInfo {
+    return WebCapabilities.cpuInfo;
+  }
```

Comment:
But is `cpuInfo` part of WebCapabilities class? All of the methods are returning if we are capable of doing something *based* on this method but not actually sharing any CPU stuff, so maybe it could be exported from index.ts as an util function instead of implementing it here

Reply:
I had a brief discussion with @brycetham about the idea that all library consumers rely on API from WebCapabilities to access user capabilities. It would make sense to include a getter for additional info about user capabilities in the WebCapabilities class. We already have cpuInfo there, so we just need to make it accessible externally.